test/dlc_tests/eks/eks_manifest_templates/kubeflow/install_kubeflow.sh

```diff
@@ -56,15 +67,24 @@ fi
 EKS_CLUSTER_NAME=${1}
 REGION_NAME=${2}
 
-echo "> Set AWS region"
-export AWS_REGION=$2
+# Check for environment variables
+if [ -z "$AWS_REGION" ]; then
+  echo "AWS region not configured"
+  exit 1
+fi
+
+# delete the manifest if operation is to upgrade kubeflow
+if [ "${OPERATION}" = "upgrade_kubeflow" ]; then
```

Comment:
This operation should exist in the `upgrade_kubeflow.sh` file

Reply:
For that, you need to use [update_kubeconfig](https://github.com/aws/deep-learning-containers/blob/master/eks_infrastructure/new_operation.sh#L152) before any cluster operation.

Reply:
OPERATION is set in build.sh and should be accessible in install_kubeflow.sh.

Reply:
lets not pollute the uninstallation in the script where the script is meant to install kubeflow. uninstall kubeflow action is specific to the upgrade operation and `upgrade_kubeflow.sh` should be the place to host it.

Reply:
I don't think it is pollution. The variable is global and is accessible in the context that it is being used.